tests/test_wave_05.py

```diff
@@ -79,35 +69,38 @@ def test_create_goal(client):
         }
     }
 
-
-@pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
     # Act
-    # ---- Complete Act Here ----
+    response = client.put("/goals/1", json={
+        "title": "Updated Goal Title",
+    })
+    response_body = response.get_json()
 
     # Assert
-    # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
-    # ---- Complete Assertions Here ----
-
-
-@pytest.mark.skip(reason="test to be completed by student")
+    assert response.status_code == 200
+    goal = Goal.query.get(1)
+    assert goal.title == "Updated Goal Title"
+    assert "goal" in response_body
+    assert response_body == {
+        "goal": {
+            "id": 1,
+            "title": "Updated Goal Title"
+        }
+    }
```

Comment:
Note that it is redundant to have all three of these asserts since the third assert covers the first two cases. I'd recommend only having the last assert (below) since it is the most specific.
```
assert response_body == {
    "goal": {
        "id": 1,
        "title": "Updated Goal Title"
    }
}
```